packages/api/src/util/cennznetExtensions.ts

```diff
@@ -0,0 +1,53 @@
+import { InterfaceTypes } from '@polkadot/types/types';
```

Comment:
I think we can reuse this type from 
https://github.com/cennznet/api.js/blob/master/packages/types/src/interfaces/extrinsic/signedExtensions/cennznet.ts

Reply:
o cool yup changed to use this type instead